src/js/actions/draganddrop.js

```diff
@@ -50,6 +50,16 @@ define(function (require, exports) {
     };
 
     /**
+    * Add many droppables at once
+    *
+    * @param {Iterable.List} list of droppable registration information
```

Comment:
Should we just rename these and get rid of the existing `register` and `deregister` functions? I.e., only have code that deals with lists of droppables?